plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,270 @@
+---
+layout: plugins/katello/documentation
+title: Use Katello with Remote Databases
+version: nightly
+---
+
+# Setup Katello with remote databases
+
+Katello can be installed with remote databases for both postgresql and mongo.
+
+## High level
+There are two ways to deploy Katello with remote databases:
+
+1. [Fresh install](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#fresh-install)
+    * prepare Postgres server with databases for Foreman and Candlepin and dedicated users owning them
+    * prepare Mongo DB with user owning the pulp_database
+    * prepare box where the Katello will be installed and make sure the databases are accessible from the box
+    * run foreman-installer with right parameters pointing to the databases
+
+2. [Migration of existing Katello installation](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#migration-of-existing-katello)
+    * prepare Postgres server with databases for Foreman a Candlepin and dedicated users owning them
+    * prepare Mongo DB with user owning the pulp_databse
+    * make sure the databases are accessible from the box where Katello is installed
+    * shut down the services except the dbs you want to move (mongod, postgresql)
+    * dump the DBs
+    * restore the DBs on remote servers
+    * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
+
+## Prepare remote Postgres
+GOAL: To use remote Postgres database with Katello we have to:
+* be able to access the databases from katello box
+* the database user we use to connect to the database needs to own the database, i.e. it can create, alter and delete the tables, indexes and constraints. Note it is not required to be able to create the database itself.
+
+### Install Postgres
+Warning: This is just minimal testing setup which is not suitable for production.
+
+Assume our postgres server has hostname `postgres.example.com`.
+
+```
+  yum -y localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  yum install postgresql-server postgresql-contrib
```

Comment:
You can remove the indenting when using the backticks. This gives better results when copy-pasting to a shell.